source/precalculus/sage/common.sage

```diff
@@ -72,4 +121,13 @@ class TBILPrecal:
     def small_complex(real_part=range(-5,6), 
                       imaginary_part=[i for i in range(-5,6) if i != 0],length=1):
         '''Generates a list of unique 2-tuples of conjugate pairs of complex (not real) numbers of the form a+bi.'''
-        return sample([(a+b*I,a-b*I) for a in real_part for b in imaginary_part],length)
\ No newline at end of file
+        return sample([(a+b*I,a-b*I) for a in real_part for b in imaginary_part],length)
+
+    @staticmethod
+    def check(LHS,operator, RHS):
+        if operator == '<': return LHS > RHS
+        if operator == '>': return LHS < RHS
```

Comment:
>
```suggestion
        if operator == '<': return LHS < RHS
        if operator == '>': return LHS > RHS
```

Reply:
Pretty sure this will fix the issue @StevenClontz spotted, but this should probably just be refactored along the lines of #257